src/doc/unstable-book/src/SUMMARY.md

```diff
@@ -78,6 +78,8 @@
 - [fmt_internals](fmt-internals.md)
 - [fn_traits](fn-traits.md)
 - [fnbox](fnbox.md)
+- [field_init_shorthand](field-init-shorthand.md)
```

Comment:
Is this line supposed to be here? This is causing the 'read only' fail that Travis is encountering